content/docs/reference/types/reference.mdx

```diff
@@ -112,4 +112,31 @@ When querying for a `post` document, the `author` key in the response will conta
 }
 ```
 
-![](https://res.cloudinary.com/forestry-demo/image/upload/v1722241391/Screenshot_2024-07-29_at_6.23.04_PM_tjxgk7.png)
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1723171157/Screenshot_2024-08-09_123906_huk6oz.png)
+
+File name will be display in the reference field selector
+
+### Search reference
+
+You can search your reference with reference selector
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1723171500/Screenshot_2024-08-09_124448_xx0g7v.png)
+
+> Note: `reference` display in the option list only support reference that have field `label` or `name`. Customize reference field with choosen field still under implementation. See [issue](https://github.com/tinacms/tinacms/issues/4707)
```

Comment:
```
> Note: The reference selector will only work if you have either the "name" or "label" field, otherwise it will default to the filename. Customizing reference fields with a chosen field is still under development. See [issue](https://github.com/tinacms/tinacms/issues/4707)
```